archivesunleashed/archivesunleashed.org
# Allows localhost serving, resolves #48

This renames the base `index.md` to `_index.md` and allows the local serving. Tested and works on @SamFritz's laptop and @ianmilligan1's workstation. We are both running on hugo v.0.44.

@ruebot do you want to do a quick check to make sure this lets you look locally on your end?

### To Test

- pull down branch `issue-48`
- `hugo serve`
- see if you can go to any page other than the landing page
